src/client.js

```diff
@@ -305,427 +305,558 @@ class ConnectionParameters {
-        queryDef = client._convertToRepresentation(queryDef, "SimpleJson");
-        const query = client.NLWS.xtkQueryDef.create(queryDef);
-        const extAccount = await query.executeQuery();
-
-        const cipher = await client._getSecretKeyCipher();
-        const type = EntityAccessor.getAttributeAsLong(extAccount, "type");
-        if (type == 3) {
-            // Mid-souring
-            const endpoint = EntityAccessor.getAttributeAsString(extAccount, "server");
-            const user = EntityAccessor.getAttributeAsString(extAccount, "account");
-            const password = cipher.decryptPassword(EntityAccessor.getAttributeAsString(extAccount, "password"));
-            return ConnectionParameters.ofUserAndPassword(endpoint, user, password, client._connectionParameters._options);
+      var queryDef = {
+        "schema": "nms:extAccount",
+        "operation": "get",
+        "select": {
+          "node": [
+            { "expr": "@id" },
+            { "expr": "@name" },
+            { "expr": "@label" },
+            { "expr": "@type" },
+            { "expr": "@account" },
+            { "expr": "@password" },
+            { "expr": "@server" }
+          ]
+        },
+        "where": {
+          "condition": [
+            { "expr": client.sdk.escapeXtk`@name=${extAccountName}` },
+            { "expr": "@type=3" }
+          ]
         }
-        throw CampaignException.CREDENTIALS_FOR_INVALID_EXT_ACCOUNT(extAccountName, type);
+      };
+        // Convert to current representation
+      queryDef = client._convertToRepresentation(queryDef, "SimpleJson");
+      const query = client.NLWS.xtkQueryDef.create(queryDef);
+      const extAccount = await query.executeQuery();
+
+      const cipher = await client._getSecretKeyCipher();
+      const type = EntityAccessor.getAttributeAsLong(extAccount, "type");
+      if (type == 3) {
+        // Mid-souring
+        const endpoint = EntityAccessor.getAttributeAsString(extAccount, "server");
+        const user = EntityAccessor.getAttributeAsString(extAccount, "account");
+        const password = cipher.decryptPassword(EntityAccessor.getAttributeAsString(extAccount, "password"));
+        return ConnectionParameters.ofUserAndPassword(endpoint, user, password, client._connectionParameters._options);
+      }
+      throw CampaignException.CREDENTIALS_FOR_INVALID_EXT_ACCOUNT(extAccountName, type);
     }
 
-}
+  }
 
-// ========================================================================================
-// ACC Client
-// ========================================================================================
+  // ========================================================================================
+  // File Uploader
+  // ========================================================================================
+
+  const fileUploader = (client) => {
+    /**
+         * Will call the SOAP method(IncreaseValue) to increament the counter
```

Comment:
I'm not too fan of those comments which do not really provide any information. Can you be a little more explicit: what is this counter about? Imagine someone reading this code who is not deeply familiar with Campaign internals. I think the comment should give some hints about why we need to increase a counter, and what this counter is about.